lib/bundles/inspec-compliance/cli.rb

```diff
@@ -56,6 +57,32 @@ def login(server) # rubocop:disable Metrics/AbcSize
       puts '', msg
     end
 
+    desc "automate SERVER --user='USER' --dctoken='DATA_COLLECTOR_TOKEN' or --usertoken='AUTOMATE_TOKEN' --ent='ENT'", 'Log in to an Automate SERVER'
```

Comment:
I had started out with just making it an extension of inspec compliance login, but then i couldn't display the desc well with the different options...which is why I went this route.  I'm up for a rename to login-automate, i think that makes sense.